Makefile

```diff
@@ -69,10 +69,9 @@ e2e-wip:
 	cd tests/e2e && npm run kakunin -- --tags @wip && cd ../..
 
 e2e-install:
-	cd tests/e2e
-	npm install
-	node ./node_modules/protractor/bin/webdriver-manager update --gecko=false
-	cd step_definitions
+	cd tests/e2e && npm install
+	node ./tests/e2e/node_modules/protractor/bin/webdriver-manager update --gecko=false
+	cd tests/e2e/step_definitions
```

Comment:
what was the problem here? do symlinks below work?

Reply:
I'm not sure.. it was this: 

```
cd tests/e2e
npm install
```

Which resulted in `make` running `npm install` in the project root, instead of in the folder it `cd`'ed to in the previous command.

Reply:
The symlinks do work, yes.